test/integration/golden_response/staging/get_stat_set_within_place/CA_County.json

```diff
@@ -9,13 +9,41 @@
             "import_name": "CensusPEP"
           }
         },
+        "geoId/06003": {
```

Comment:
Note the added places. The new implementation guarantees full coverage of all the places

Reply:
nice!

Reply:
Nice. So the old implementation had fewer places because of date limitation?

Reply:
For this case, it's a bug from old implementation. The data is correctly picked but the source cohorts was picked by source ranking. Instead it should have picked the source with the most places.

In the file below (get_stat_set_within_place/US_State.json), the missing data is due to CensusPEP not having geoId/11 data in 2019. It should have use both sources.